vbfml/scripts/err_analysis

```diff
@@ -455,5 +489,313 @@ def compare_hist(ctx, input_dirs: str, number_epochs: int, save_name: str) -> No
     plt.savefig(pjoin(outdir, f"comparison_val_loss_{save_name}.pdf"))
 
 
+@cli.command()
+@click.pass_context
+@click.option(
+    "-i",
+    "--input-dir",
+    required=True,
+    help="path to the prediction directory of a model",
+)
+def roc(ctx, input_dir: str) -> None:
+    """
+    compute the roc curve of the model. /!/ input_dir point at the prediction_dir without a '/' at the end.
+    only compute and write in pkl file so it allows to use plot-roc on several model to compare in in one plot
+    """
+
+    # load model and prediction pickle file
+    loader = TrainingLoader(os.path.dirname(input_dir))
```

Comment:
To get rid of the trailing `/` problem, one method I found useful is to do: `input_dir.rstrip("/")`. This way, if user specifies a directory name with a trailing `/`, it's stripped from the string. If not, string is not changed at all, so it should work both ways. 

So to make this function work both ways, we can update this line to be:
```
loader = TrainingLoader(os.path.dirname(input_dir.rstrip('/')))
```
And you can remove the note about the trailing `/` from the docstring (always good to test it first though hahah). In general, if you encounter similar problems somewhere else in the code, I think this is a good way to mitigate it.

Reply:
good to know thank you very much
I used it